scaleway/resource_lb_backend.go

```diff
@@ -374,6 +375,9 @@ func resourceScalewayLbBackendRead(ctx context.Context, d *schema.ResourceData,
 	_ = d.Set("health_check_http", flattenLbHCHTTP(backend.HealthCheck.HTTPConfig))
 	_ = d.Set("health_check_https", flattenLbHCHTTPS(backend.HealthCheck.HTTPSConfig))
 
+	sendProxy := backend.SendProxyV2
+	_ = d.Set("send_proxy_v2", &sendProxy)
```

Comment:
Should we have a flatten function on this?

Reply:
not yet exist. Going to add one.